### PR TITLE
TM-7311: Add docs for new Custom Form tracking commands

### PR DESCRIPTION
**Pull request by musialp**
Provide documentation for new commands used for tracking Custom Forms

### CODE REVIEW

File: consent_manager/js_api/commands.rst [hunk 1]
```diff
@@ -243,3 +243,136 @@ Code::
...
+.. versionadded:: 15.3
+Command used to track Consent Form main view (automatic view, when user enters the website for the first time).
+
+Code::
+
+    ppms.cm.api('trackMainFormView', onFulfilled, onRejected);
+
+.. function:: onFulfilled()
+
+    The fulfilment handler callback.
```
Comment (zawias-pro) [inline consent_manager/js_api/commands.rst:258]:
```suggestion
    The fulfillment handler callback.
```
this with one `l` is BE

**Review by zawias-pro — COMMENTED** [review]
There is 
> After successful, Consent Manager internally sends only to stats endpoint an information that user has seen consents.

in set initial consents command description. Should we create a task to update this doc once https://piwikpro.atlassian.net/browse/TM-7309 is done.

File: consent_manager/js_api/commands.rst (hunk 1 above)
Comment (musialp) [reply consent_manager/js_api/commands.rst]: Fixed

**Comment by musialp** [comment]
> There is
> 
> > After successful, Consent Manager internally sends only to stats endpoint an information that user has seen consents.
> 
> in set initial consents command description. Should we create a task to update this doc once https://piwikpro.atlassian.net/browse/TM-7309 is done.

Added a task for it.

**Review by zawias-pro — APPROVED** [review]

**Pull request merged**